### PR TITLE
test(DocSearch): Add tests for constructor

### PR DESCRIPTION
**Pull request by pixelastic**

### CODE REVIEW

**Comment by pixelastic** [comment]
Hmm weird, I'm having a look.

**Comment by pixelastic** [comment]
Fixed, ready for review.

**Comment by pixelastic** [comment]
Can I haz review?

plz

**Comment by ElPicador** [comment]
LGTM

File: test/DocSearch-test.js [hunk 1]
```diff
@@ -1,107 +1,257 @@
...
+      // Given
+      let options = defaultOptions;
+
+      // When
+      new DocSearch(options);
+
+      // Then
+      expect(algoliasearch.addAlgoliaAgent.calledOnce).toBe(true);
+    });
+    it('should instanciate autocomplete.js', () => {
```
Comment (proudlygeek) [inline test/DocSearch-test.js]: Instantiate :wink:

**Pull request merged**